inc/endpoints/class-components-endpoint.php

```diff
@@ -236,9 +239,12 @@ public function build_query() {
 					// This is a verbose page match, let's check to be sure about it.
 					$page = get_page_by_path( $matches[ $varmatch[1] ], OBJECT, $page_type );
 					if ( ! $page ) {
+						$is_404 = true;
 						continue;
 					}
 
+					$is_404 = false;
```

Comment:
I could be missing something or not following indentation correctly, but if we default this to `false` on line `219` do we need to set it here too?

Reply:
Ah but this _would_ be required if we're flipping back and forth during the `foreach` loop... I'll leave it up to you 😸